engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -169,11 +169,8 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
     // TODO: one day the main light (sun/moon) should be just another light in the scene.
     private void initMainDirectionalLight() {
         mainDirectionalLight.lightType = LightComponent.LightType.DIRECTIONAL;
-        mainDirectionalLight.lightColorAmbient = new Vector3f(1.0f, 1.0f, 1.0f);
-        mainDirectionalLight.lightColorDiffuse = new Vector3f(1.0f, 1.0f, 1.0f);
-        mainDirectionalLight.lightAmbientIntensity = 1.0f;
-        mainDirectionalLight.lightDiffuseIntensity = 2.0f;
-        mainDirectionalLight.lightSpecularIntensity = 0.0f;
```

Comment:
Is the idea here that eventually we'll get block/material-level specular control, so that grass does not have specular reflections?